packages/protocol-kit/src/types/safeProvider.ts

```diff
@@ -43,7 +43,7 @@ export type PasskeyClient = Client<
   WalletActions<Chain | undefined, Account> & PasskeyActions
 >
 
-export type ExternalSigner = WalletClient<Transport, Chain | undefined, Account> | PasskeyClient
+export type ExternalSigner = WalletClient<Transport, Chain | undefined, Account>
```

Comment:
Why did you change this type?

https://github.com/safe-global/safe-core-sdk/blob/dc98509f15f42f04d65edcbada3bcf1f61f1154a/packages/protocol-kit/src/SafeProvider.ts#L172